### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by Anmol2001**
I have center aligned and improved the participants column for any screen as well as fixed the overlapping of participant's name and messages in chat by changing position  attribute. This is my first pull request concerned with the issue #61  👍.
### **Before**
The chat messages were overlapping with the participants column and column was not properly aligned.
image
### **After**
Both the issues are fixed and now the chat message being displayed  at top of screen  slides downwards on opening the participant's column.
aaaaaaAnnotation 2020-10-01 154321

### CODE REVIEW

**Comment by abhigyank** [comment]
Can you put a before and after screenshot?

**Comment by Anmol2001** [comment]
yeah sure just give me an hour after quiz i ll surely do it.

File: public/style.css [hunk 1]
```diff
@@ -326,11 +326,10 @@ body {
...
   background-color: rgba(0, 0, 0, 0.08);
-  position: fixed;
+  position: sticky;
   top: 80px;
-  right: 20%;
-  height: 0%;
-  width: 23%;
+  margin: auto;
+  width: 60%;
   transition: .5s;
```
Comment (LalitNM) [inline public/style.css]: The initial height of container should be zero. Right? Everything else looks fine.

Comment (Anmol2001) [reply public/style.css]: @LalitNM  yes initial height should be zero.

**Comment by abhigyank** [comment]
@Anmol2001 in the 1st screenshot of After, it looks like the message is going below the input box. Can you fix that and update?

**Comment by Anmol2001** [comment]
> @Anmol2001 in the 1st screenshot of After, it looks like the message is going below the input box. Can you fix that and update?

@abhigyank  sure ,i will try.

**Comment by LalitNM** [comment]
@Anmol2001 your branch has so many conflicts, either you reset this branch and then rebase with upstream or open another PR after closing this one.

**Comment by LalitNM** [comment]
closing in favor of fa4c0c369d52be122dd1cc649cb93c3aee099510

**Pull request closed without merging**